rules/windows/registry/registry_set/registry_set_disable_windows_event_log_access.yml

```
@@ -0,0 +1,35 @@
+title: Windows Event Log Access Tampering Via Registry
+id: ba226dcf-d390-4642-b9af-b534872f1156
+status: experimental
+description: |
+    Detects changes to the Windows EventLog channel permission values. It focuses on changes to the Security Descriptor Definition Language (SDDL) string, as modifications to these values can restrict access to specific users or groups, potentially aiding in defense evasion by controlling who can view or modify a event log channel. Upon execution, the user shouldn't be able to access the event log channel via the event viewer or via utilities such as "Get-EventLog" or "wevtutil".
+references:
+    - https://www.atomicredteam.io/atomic-red-team/atomics/T1562.002#atomic-test-8---modify-event-log-channel-access-permissions-via-registry---powershell
+    - https://www.youtube.com/watch?v=uSYvHUVU8xY
```

```suggestion
references:
    - https://www.atomicredteam.io/atomic-red-team/atomics/T1562.002#atomic-test-8---modify-event-log-channel-access-permissions-via-registry---powershell
    - https://www.youtube.com/watch?v=uSYvHUVU8xY
    - https://learn.microsoft.com/en-us/windows/win32/secauthz/security-descriptor-definition-language
```
Just add the link to SDDL